R/loadReg2.R

```diff
@@ -147,7 +148,8 @@ loadReg2 <- function(load.reg,
   }
   
   # Get the metadata, which we'll need to do the following data checks
-  meta <- getMetadata(load.reg)
+  #fill the site.id field, since rloadest only has a site.name equivalent
+  meta <- updateMetadata(getMetadata(load.reg), site.id = site.id)
```

Comment:
Cool, I like where this is headed. This is inspiring me to make a new issue for how to allow the user to add any of the metadata pieces that aren't already collected in the `loadReg` call, but I think what you've done is the correct scope for our ANA needs, so let's keep it just like this for the time being.